### ISSUE TITLE
Element.getBoundingClientRect()

### ISSUE THREAD

**Issue by ghost** [sharplispers/cxml#8]
Is it possible to call DOM's `getBoundingClientRect()` on an Element which i get by:

    (aref (dom:get-elements-by-tag-name
           (cxml:parse-file "./test.svg" (cxml-dom:make-dom-builder))
           "path") 0)

This returns a `#<RUNE-DOM::ELEMENT path {10042A87A3}>` from the SVG file test.svg.

**Comment by scymtym** [comment]
I don't think that is implemented. cxml only covers the XML document model, not the HTML DOM or SVG.

**Comment by slyrus** [comment]
What about:
```
(aref (dom:get-elements-by-tag-name
       (cxml:parse-file "./test.svg" (cxml-dom:make-dom-builder))
       "svg:path") 0)
```
